src/components/LoginSignUp/LoginSignUpWrapper.tsx

```diff
@@ -78,25 +80,16 @@ const LoginSignupWrapper = ( {
     return unsubscribe;
   }, [navigation] );
 
-  const fitContentWithinScreenStyle = { height: windowHeight * 0.85 };
+  // const fitContentWithinScreenStyle = { height: windowHeight * 0.85 };
```

Comment:
This seems to be still required because now there is too little spacing between the "A global community ..." header and the first text input.

Reply:
ahhhh, I missed reverting this from testing. It's restored now.